### DISCUSSION TITLE
Kpn IPTV worked for 1 day un UDM an now fails

### DISCUSSION THREAD

**Discussion by Gsxr1001** [fabianishere/udm-iptv#283] [Q&A]
Kpn IPTV worked for 1 day on a unifi udm.

This is the udm-iptv diagnose:

 Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth4
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br4
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@208]: if_mtu = 1500
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@214]: if_falgs = 4163
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@178]: iptv's ipv4 address is 10.226.10.102
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@195]: if_index = 1026
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@208]: if_mtu = 1500
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: DEBUG[init_interface@214]: if_falgs = 4163
Oct 26 19:59:14 UDM-Bakhuis udm-iptvd[321744]: ERROR[main@619]: exiting.........
Oct 26 19:59:14 UDM-Bakhuis IMPROXY[321744]: ERROR[main@619]: exiting.........
Oct 26 19:59:14 UDM-Bakhuis systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Oct 26 19:59:14 UDM-Bakhuis systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

What is going wrong
Can someone give me some help?

**Comment by Multied1** [comment]
Hi Gsxr1001,

Do you use a separate vlan for IPTV? And did you try to run the script again?
What unifiOS and application do you run?
